### DISCUSSION TITLE
PI Zero 2 with pHAT v2.6+ & Genmon install keeps Crashing

### DISCUSSION THREAD

**Discussion by TEBrewMeister** [jgyates/genmon#1185] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

Genmon
Version V1.19.04

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

BLUF: The GENMON system periodically just disappears. The only solution is to remove the power and let it reboot.  Then all is good again till the next disappearance. The RPI Zero2 is installed in a waterproof box inside the generator, and I connect via wifi to a reserved IP.  I thought perhaps it was overheating, but temps have cooled and it still drops off. Sometimes it will be OK for 7-10 days and then disappear, other times it will go out in a day or two. 

I've changed SD CARD, no difference.  I've attempted to order a replacement PI ZERO 2 with headers, but no luck yet.

My PI ZERO 2 with Genmon is my first Raspberry PI adventure. Since installing GENMON on my 2023 Generac/Honeywell 22KW system I've also set up a PIHOLE with a RPI Zero 2.  All that to say I don't know what I don't know... and I've tried to ferret out what might be the problem with no luck. 

Appreciate any suggestions.

**Comment by jgyates** [comment]
When it disappears, can you ping or log into ssh, or does the web interface not respond or both? One thing you can try is to wait until it disappears then physically turn the generator Off, then back to Auto and see if you get notified even if you can not access the web interface.

These are just guesses at this point: This could be a power problem or an issue with your router?

Pi Zeros can be a little more sensitive to non ideal power. How are you powering the pi zero? 

Do you have an Asus router? If you do this could be a problem with the "AIProtection".  If you do you may have to disable some of the AIProtedtion features as they interfear with / block  web servers on your local network. More info on this can be found in [past issues here](https://github.com/jgyates/genmon/issues?q=is%3Aissue+label%3ARouter+asus).
